modules/users/server/controllers/admin.server.controller.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+var path = require('path'),
+	mongoose = require('mongoose'),
+	User = mongoose.model('User'),
+	errorHandler = require(path.resolve('./modules/core/server/controllers/errors.server.controller'));
+
+/**
+ * Show the current user
+ */
+exports.read = function (req, res) {
+	res.json(req.model);
+};
+
+/**
+ * Update a User
+ */
+exports.update = function (req, res) {
+	var user = req.model;
+
+	//For security purposes only merge these parameters
+	user.firstName = req.body.firstName;
+	user.lastName = req.body.lastName;
+	user.roles = req.body.roles;
+
```

Comment:
Should we be updating the displayName here as well? 

Perhaps, it's a decision for the developer user of this project. I can see some developers preferring to keep this data in the hands of the app user. I'm fine with this method updating the displayName; after all, it is an admin that's doing the update. They always know best.